backend/ontario/ontario/__init__.py

```diff
@@ -41,41 +46,59 @@ def __init__(self, context: ImageContext):
 
     def load_from_file(self, path: str) -> "ImageBuilder":
         """
-        Loads an image from a file.
+        Loads an image file to create a source node.
+        Currently only supports .png, .jpg, and .exr
         """
 
-        node = Gegl.Node()
-        node.set_property("operation", "gegl:load")
-        node.set_property("path", path)
-        self.__parent.add_child(node)
+        # get file extention
+        split_ext: tuple = os.path.splitext(path)
+        file_ext: str = split_ext[-1]
+        file_ext = file_ext.lower()
+
+        # add node as a child to the image context
+        if file_ext == '.png':
```

Comment:
Ontario-Web currently stores images as webp, is there a way to support webp here or should I move to one of these formats?

Reply:
yes, I will add this now